server/routes/admin/licence.ts

```diff
@@ -128,7 +128,7 @@ export = (licenceService, signInService, prisonerService, audit, roNotificationH
         if (error) {
           req.flash('errors', { notifyError: error.message })
         }
-        return res.redirect(`/admin/licences/${bookingId}`)
+        return res.redirect(`/admin/licenceSearch/${bookingId}`)
```

Comment:
fix issue where notify to RO would go back to an empty licence page, now it goes to the search page